orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/PreviousImageRollback.groovy

```diff
@@ -97,6 +97,8 @@ class PreviousImageRollback implements Rollback {
       credentials                  : parentStageContext.credentials,
       cloudProvider                : parentStageContext.cloudProvider,
       delayBeforeDisableSec        : delayBeforeDisableSeconds ?: 0,
+      targetGroups                 : parentStageContext.targetGroups,
```

Comment:
what if this deploy changes the `targetGroup` or `securityGroup` to something new, shouldn't we restore it to the original state?

Reply:
Actually, looks like only `deck` uses `PREVIOUS_IMAGE`. Do you have a corresponding `deck` PR for this as well?

Reply:
Will fire up a deck PR as a follow up to re-enable Titus previous image rollbacks and ensure target groups are passed through the request.